standard/statements.md

```diff
@@ -456,8 +456,12 @@ local_function_header
     ;
 
 local_function_modifier
-    : 'async'
-    | unsafe_modifier   // unsafe code support
+    : ref_local_function_modifier
+    | 'async'
+    ;
+
+ref_local_function_modifier
```

Comment:
I chose to represent it this way rather than duplicate the *unsafe_modifier* in both nodes to make it easier to update when `static` local functions are added.